### ISSUE TITLE
Move to using tensorflow-nufft

### ISSUE THREAD

**Issue by chaithyagr** [zaccharieramzi/fastmri-reproducible-benchmark#174]
Recently, tensorflow-nufft came up, which is based out of cufiNUFFT, which is known to be computationally and memory efficient.

I will proceed with initial tests for this and get a PR to completely move away from `tfkbnufft`

**Comment by zaccharieramzi** [comment]
Makes sense, after we have a test that confirms this first well informed opinion.

**Comment by zaccharieramzi** [comment]
@chaithyagr can you use this issue to keep track of the progress made on the convergence using `tensorflow-nufft`?

**Comment by chaithyagr** [comment]
I think that is fixed in tensorflow nufft.. we have minor issues still there in graph mode which I'm actively fixing. But it's tracked there.
